src/components/ListLinks.js

```diff
@@ -0,0 +1,28 @@
+import style from 'styled-components';
```

Comment:
Where is this component used? If not please remove it

Reply:
Deleted component ListLinks.js